tests/unit/resources/mint_test.py

```diff
@@ -0,0 +1,60 @@
+"""Test Mint resource component."""
+from unittest import mock
+
+import pytest
+
+from magic_admin.resources.mint import Mint
+
+
+class TestMint:  # noqa: WPS306
+    """Test Mint resource component."""
+
+    quantity = 2
+    destination_address = '0x3c15B0e0e00A9edD2Be824064f9C9C29fc136C4E'
+    token_id = 1
+
+    @pytest.fixture(autouse=True)  # noqa: PT004
+    def setup(self):
+        """Prepare test case."""
+        self.mint = Mint()  # noqa: W0201
+        self.contract_id = 'bsdjfkn-sjknfskn-kjsnf'  # noqa: W0201
```

Comment:
Nit: since these are new tests, this gives us an opportunity to write tests that adhere to backend testing guidelines: https://www.notion.so/magiclabs/Backend-Testing-Style-Guidelines-3caaf40a52e948ebaf17f45a4838fdce?pvs=4

A lot of the tests in this repo conform to the old pattern so it's not entirely necessary (hence the nit)

Reply:
If you prefer to keep as is, I would prefer to move `self.contract_id = 'bsdjfkn-sjknfskn-kjsnf'` to the class-level with the other attributes for consistency.